src/components/CurrentTerm.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import HierarchyItem from "./HierarchyItem";
+import { Box, Paper, Typography, withStyles } from "@material-ui/core";
+import { TermAccordionProps } from "./TermAccordion";
+
+const CurrentTermBox = withStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.secondary.main,
+    marginBottom: -1,
+    minHeight: 56,
+    display: "flex",
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(7),
+    alignItems: "center",
+    paddingTop: 12,
+    paddingBottom: 12,
```

Comment:
You can use something like `padding: theme.spacing(1.5, 7, 1.5, 2)` to cover all the padding rules.